### PR TITLE
Zoma/integrating

### PR DESCRIPTION
**Pull request by coldplaying42**
Update the code of integration and dialogGenerator to botbuilder-lg 4.8.0-preview

### CODE REVIEW

File: packages/dialog/README.md [hunk 1]
```diff
@@ -112,3 +112,26 @@ bf dialog:verify test/**/*.dialog
 ```
 
 _See code: [src/commands/dialog/verify.ts](https://github.com/microsoft/botframework-cli/blob/v0.0.0/src/commands/dialog/verify.ts)_
+
+## `bf dialog:integrate [FILE]`
```
Comment (chrimc62) [inline packages/dialog/README.md]: >`bf dialog:integrate [FILE]` [](start = 3, length = 28)

Doesn't this go away and need to be included in above?

File: packages/dialog/src/library/dialogGenerator.ts [hunk 2]
```diff
@@ -162,27 +162,27 @@ async function processTemplate(
         if (ref) {
             outPath = ppath.join(outDir, ref.relative)
         } else {
-            let template = await findTemplate(templateName, templateDirs, scope.locale)
-            if (template !== undefined) {
+            let lgfile = await findTemplate(templateName, templateDirs, scope.locale)
+            if (lgfile !== undefined) {
```
Comment (chrimc62) [inline packages/dialog/src/library/dialogGenerator.ts]: >lgfile [](start = 16, length = 6)

Please keep this as template.  It is not always an lgFile.

File: packages/dialog/src/commands/dialog/integrate.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        help: flags.help({ char: 'h' }),
+        oldPath: flags.string({ char: 'o', description: 'path of old assets ', required: true }),
+        newPath: flags.string({ char: 'n', description: 'path of new assets ', required: true }),
+        mergedPath: flags.string({ char: 'm', description: 'path of merged assets ', required: true }),
+        locale: flags.string({ char: 'l', description: 'locale', required: true }),
+        verbose: flags.boolean({ description: 'output verbose logging of files as they are processed', default: false }),
+    }
+
+    static examples = [
+        '$ bf dialog:integrete aaa -o /bbb -n /ccc -m /ddd -l en-us'
```
Comment (chrimc62) [inline packages/dialog/src/commands/dialog/integrate.ts]: >integrete [](start = 21, length = 9)

spelling, but I thought you were working on full integration?

**Comment by cleemullins** [comment]
This PR has has merge conflicts, is not up-to-date with master, and is simply too large (162 files) to review. Closing. 

Please refactor and resubmit.

**Pull request closed without merging**